ToDoTakeTWo/app/src/main/java/com/korbkenny/todotaketwo/ListActivity.java

```diff
@@ -0,0 +1,167 @@
+package com.korbkenny.todotaketwo;
+
+import android.content.Context;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.ActionBar;
+import android.support.v7.app.AlertDialog;
+import android.support.v7.app.AppCompatActivity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ListView;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+ * Created by KorbBookProReturns on 10/20/16.
+ */
+
+public class ListActivity extends AppCompatActivity{
+
+    private TextView mTitle;
+    public BigOldListOfLists semiBigListofListsIguess;
+    private Button mButton;
+    private ListView mListOfItems;
+    public ArrayAdapter<ToDoItem> mArrayAdapter2;
+    private ArrayList<ToDoItem> mToDoItemLists;
+    final Context context = this;
+    private ToDoList current;
+    private Button mBackButton;
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_list);
+
+        //GET INTENT
+        String title = getIntent().getStringExtra("Title");
+        final int index = getIntent().getIntExtra("Index",-1);
```

Comment:
Don't forget to handle the case where `index` ends up with value `-1` due to some unforeseen error. Later you pass it to the list of lists' `get()` method as a parameter, so if it was `-1` then at that point it would crash your app. It would be better to check for that possibility now and then log it and/or `finish()` the activity.